healthScore/models.py

```diff
@@ -56,6 +56,12 @@ def create_staff(self, email=None, password=None, **extra_fields):
         extra_fields.setdefault("is_patient", False)
         return self._create_user(email, password, **extra_fields)
 
+    def create_healthcare_worker(self, email=None, password=None, **extra_fields):
```

Comment:
I didn't see this function being used. Is it required?

Reply:
I think down the road it'll be needed for when healthcare_admins are able to add healthcare workers to the system. Its not being used right now though. Let me know if you want me to take it out for now.

Reply:
Ohh got it, makes sense.
We have a boolean field `admin` in `HospitalStaff` table which can be used to make this distinction. That's why I didn't add a specific key for doctors in the `Users` table.

But please let me know if I am missing something, if it is better to have this key in the Users table - we can get rid of the admin key then

Reply:
Ahh I see. It might make things a bit easier if it were in the users table, that way the create_healthcare_workers function would be able to work a bit easier. We wouldn't have to reference the other db to find that flag if it were just in Users. Let me know if you think that wouldn't be a problem though

Reply:
Agreed!